repos/system_upgrade/el7toel8/libraries/persistentnetnames.py

```diff
@@ -0,0 +1,37 @@
+import pyudev
+
+from leapp.models import PCIAddress, Interface
+
+udev_context = pyudev.Context()
+
+def physical_interfaces():
```

Comment:
Add please docstring for every function in shared library regarding the https://github.com/oamg/leapp-repository/blob/master/CONTRIBUTING.md#code-style

These functions are visible by all actors so we would like to have them documented.